lib/src/main/java/com/orange/ods/compose/text/OdsText.kt

```diff
@@ -68,7 +65,7 @@ fun OdsText(
     OdsText(
         text = text,
         modifier = modifier,
-        color = displaySurface.themeColors.onSurface.enable(enabled = enabled),
+        color = LocalContentColor.current.enable(enabled = enabled),
```

Comment:
I think we can now remove all the `Color.Unspecified` values that we set to fix some text issues.

For instance this one in the bottom navigation is not necessary anymore because the color is not overridden by the display surface:

```kotlin
OdsText(
    text = label,
    // Set color to Unspecified otherwise the colors parameter of BottomNavigationItem is overridden
    // by the default value of the displaySurface parameter of OdsText
    color = Color.Unspecified,
    maxLines = 1,
    overflow = TextOverflow.Ellipsis,
    style = OdsTextStyle.BodyS
)
```